frontend/src/pages/index.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import Layout from '../components/layout';
-import List from '../components/aeppslist';
+import AeppList from '../components/aepps-list';
```

Comment:
File name to match name of the component
AeppList - aepp-list.js
or
AeppsList - aepps-list.js